js/main.js

```diff
@@ -1,6 +1,36 @@
 'use strict';
 
 // anime.js利用
+/* ハンバーガーメニュー */
+const menuOpen = document.getElementById('open');
+const menuClose = document.getElementById('close');
+const overlayMenu = document.querySelector('.header__overlay');
+const backgroundMask = document.getElementById('blackBg')
+
+menuOpen.addEventListener('click', function (e) {
+  openOverlayMenu();
+});
+
+menuClose.addEventListener('click', function (e) {
+  closeOverlayMenu();
+});
+
+backgroundMask.addEventListener('click', function (e) {
+  closeOverlayMenu();
+});
+
+
+// 上で呼び出す各関数
+function openOverlayMenu() {
+  overlayMenu.classList.add('header__overlay--open');
```

Comment:
```suggestion
const openOverlayMenu = () => {
  overlayMenu.classList.add('header__overlay--open');
```

関数はアロー関数で書くようにしていくと後々困りません
自分の実体験でアロー関数みにくいので後回しにしてたら痛い目にあったのでToT

Reply:
実は元々最初はアローで書いてたんですが、他の箇所にあわせてこっちにかえちゃってました。
書き方の違いだけではないんですね・・・ググってみます！
ご指摘ありがとうございます(=ﾟωﾟ)ﾉ

Reply:
@hashibadaiki 
修正完了しました！
他の箇所でも書き換えた方がいいかなと思う関数はアロー関数へ変更しています。
ご確認をお願いいたします。